images/linux/scripts/installers/1804/basic.sh

```diff
@@ -97,6 +97,9 @@ apt-get install -y --no-install-recommends xvfb
 echo "Install libgtk"
 apt-get install -y --no-install-recommends libgtk-3-0
 
+echo "Install tk"
+apt install -y tk
```

Comment:
you mention that this is used when building python in hosted toolcache? why is it not available here already?

Also, can we add some validation to make sure this is in fact functional?

Reply:
@alepauly , It is not enough to add this package to HostedToolCache. This package is needed in runtime to use Tkinter packages so we have to add it to the image.
Also we will build canary test for validation

Reply:
@sergey-akhalkov . we have a successful builds and testing of all Python versions on Ubuntu 16/18.
Please merge this PR